meterfs/node.c

```diff
@@ -45,6 +46,7 @@ int node_add(file_t *file, id_t id, rbtree_t *tree)
 {
 	node_t *r;
 
+	assert(id != 0);
```

Comment:
This `assert` is IMHO hard to justify - why is it needed? `id = 0` won't cause `lib_rb*` malfunction in any way, so it does not protect against anything and in the proposed version of the code it is actually impossible to generate `id == 0` input.

I think you actually want this assert in `node_getById` instead.